app/routes/application.js

```diff
@@ -24,11 +25,17 @@ export default TravisRoute.extend(BuildFaviconMixin, {
       this.afterSignOut();
     });
 
-    this.router.on('routeDidChange', () => {
-      if (config.gaCode) {
-        _gaq.push(['_trackPageview', location.pathname]);
-      }
-    });
+    if (config.metricsAdapters.length > 0) {
+      const { metrics, raven, router } = this;
+      router.on('routeDidChange', () => {
+        try {
+          const { currentURL: page } = router;
+          metrics.trackPage({ page });
+        } catch (err) {
+          raven.logException('Metrics error');
```

Comment:
Hmm.. I'm wandering would it send errors to Sentry when tracking is disabled by adblockers? In that case we may have a loooot of the errors that we cannot avoid.

Reply:
Ahhh, that's a good point! Maybe I'll remove this for now, and we can reintroduce it in the future if needed